server/pbench/bin/index-pbench

```diff
@@ -78,10 +124,96 @@ def get_hosts(config):
 
 _op_type = "create"
 
+def es_template(es, config):
```

Comment:
This is problematic and one of the reasons we should not merge this right now.  This method only deals with one mapping file for results, and we need one for the tools data now as well.